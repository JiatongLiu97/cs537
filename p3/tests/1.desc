schedtest user app test - invoking user application schedtest 1 14 2 3 100
