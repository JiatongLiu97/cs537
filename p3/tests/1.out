$ schedtest 1 14 2 3 100
14 3
